go.mod

```diff
@@ -8,29 +8,28 @@ require (
 	github.com/gonum/graph v0.0.0-20190426092945-678096d81a4b
 	github.com/openshift/api v0.0.0-20210423140644-156ca80f8d83
 	github.com/openshift/build-machinery-go v0.0.0-20210209125900-0da259a2c359
-	github.com/spf13/cobra v1.4.0
+	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
 	google.golang.org/api v0.57.0
-	k8s.io/api v0.25.4
-	k8s.io/apimachinery v0.25.4
-	k8s.io/apiserver v0.25.4
+	k8s.io/api v0.26.1
```

Comment:
I'm not sure we should bump to 0.27.2 or stay at 0.26.1

Reply:
This PR is quite big and 4.13 sits on 1.26, so I would keep it as is.